src/SvgIcon/SvgIcon.js

```diff
@@ -0,0 +1,72 @@
+// @flow weak
+
+import React, { PropTypes } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+import { easing } from '../styles/transitions';
+
+export const styleSheet = createStyleSheet('SvgIcon', (theme) => {
+  return {
+    SvgIcon: {
```

Comment:
Quite right, my mistake; however I notice some components use `componentName` where others use `root`. Is there a subtle difference in their structure, or an inconsistency in naming? 

If it's the latter I'll gladly standardise them in a separate PR.